### PR TITLE
Feature update property

### PR DESCRIPTION
**Pull request by ghost**
Added the client side for update property feature. Will be changed to be only accessible by agents in the next update.

### CODE REVIEW

File: scripts/components/OwnerPropertyDetails.jsx [hunk 1]
```diff
@@ -38,6 +39,11 @@ var OwnerPropertyDetails = React.createClass({
           <div style={style.address}>
             <div>{street} {suburb}, {postcode}</div>
           </div>
+          <div style={style.address}>
+            <UpdatePropertyForm 
+                propertyDetailsUpdated={this.getPropertyDetails}
```
Comment (nirrek) [inline scripts/components/OwnerPropertyDetails.jsx]: `this.getPropertyDetails` is not a defined method...

File: scripts/components/UpdatePropertyForm.jsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          return;
+        }
+
+        // Clear the form
+        this.setState({
+          tenantEmail: '',
+          image: '',
+        });
+
+        this.props.inspectionReportAdded();
```
Comment (nirrek) [inline scripts/components/UpdatePropertyForm.jsx]: `this.props.inspectionReportAdded()` is not a property that is passed to the component. This means that the submit callback is not occurring, and the form is also not being dismissed.

**Comment by nirrek** [comment]
This feature was non-functional btw. You were passing a non-existent function `getPropertyDetails` to the `<UpdatePropertyForm>` component. You were also invoking a non-existent function `inspectionReportAdded` on form submission. This meant that the form was not being dismissed, nor was any updated data being fetched.

A pull request should be a fully functional feature that is ready for deployment. This means you should have properly tested your feature. Please don't just submit a PR for half-finished stuff.

**Comment by nirrek** [comment]
Stuff that needs doing:
- [ ] All details of the property need to be updateable.
- [ ] All existing data for the property needs to be prepopulated in the form, and should be submitted to the server.
- [ ] Clientside validation to ensure all fields have acceptable values prior to form submission.
- [ ] Cleanup state correctly when the dialog is closed. Eg. If there is an image size error, and you close/reopen dialog it is still sitting there.
- [ ] As with the add property form, there should be immediate feedback to the user if they enter an email for which a user does not exist.

**Comment by ghost** [comment]
Should be good now let me know about any issues, also it was already merged with master when I made my commits so it should be working form that front.

**Comment by nirrek** [comment]
@jListo Again, same as the comment I left on your server PR, there are no new commits from you that address the issues. You probably have only made commits to your local repository.

**Pull request merged**